pkg/controller/template/test_data/templates/libvirt/master/units/kubelet.service

```diff
@@ -6,34 +6,9 @@ contents: |
   After=docker.service kubelet-workaround.service
 
   [Service]
-  Environment="KUBELET_IMAGE=openshift/origin-node"
   EnvironmentFile=-/etc/kubernetes/kubelet-workaround
 
-  ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
```

Comment:
what causes these changes? is that because the hyperkube image is updated?

Reply:
@yifan-gu this is from
https://github.com/coreos-inc/tectonic-operators/pull/447

Reply:
yes please. run kubelet on RHCOS, not containerized.